javascript/code-challenges/linkedList/README.md

```diff
@@ -0,0 +1,12 @@
+### Linked List 
+Linked List Implementation 
+
+By Sowmya Billakanti 
+
+### Challenge
+- Create Nodes
+- Create a head node
+- Define methods - insert, includes, toString
+
```

Comment:
I would like to see a little more in this readme, I would add a section that you can call API where you list each method that you are creating with a description of what they are doing